Stock_Alert/errorhandling.py

```diff
@@ -0,0 +1,32 @@
+import requests
+import json
+
+class ErrorHandling:
+    def __init__(self, endpoint):
+        self.endpoint = endpoint
+
+
+    def get_hold_of(self, **kwargs):
+        try:
+            response = requests.get(self.endpoint, params=kwargs)
```

Comment:
```suggestion
            response = requests.get(self.endpoint, params)
```

Reply:
Changed it according to your suggestion.